src/profiles.jl

```diff
@@ -603,8 +609,9 @@ end
 """
     keys(commp::CommunityProfile, sample::AbstractString)
 
-Return an iterator over all keys of samples in a CommunityProfile. collect(keys(a)) returns an array of keys. When the keys are stored internally in a hash table, as is
-the case for `Dict`, the order in which they are returned may vary. But `keys(a)` and `values(a)` both iterate a and return the elements in the same order.
+Return an iterator over all keys of samples in a CommunityProfile. collect(keys(a)) returns an array of keys. 
```

Comment:
but backticks around `collect(keys(a))`

Reply:
This is getting the keys of the _metadata_ attached to one `sample`, not to the keys of all samples